.golangci.yaml

```diff
@@ -1,6 +1,26 @@
+version: "2"
 linters:
   enable:
     - gosec
-
+  disable:
+    - staticcheck
+  exclusions:
+    generated: lax
+    presets:
+      - comments
+      - common-false-positives
+      - legacy
+      - std-error-handling
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
+formatters:
+  exclusions:
+    generated: lax
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
```

Comment:
Let's try removing the entire `formatters` block to pass all files to the formatters. Using the defaults should be the best on the long run, but first, let's see if it works. 😅